src/apps/service/apps/jobs/resubmit.clj

```diff
@@ -0,0 +1,86 @@
+(ns apps.service.apps.jobs.resubmit
+  (:use [slingshot.slingshot :only [try+ throw+]])
+  (:require [apps.clients.data-info :as data-info]
+            [apps.service.apps.jobs.submissions :as submissions]
+            [apps.service.apps.jobs.submissions.async :as async]
+            [cheshire.core :as cheshire]
+            [clojure.string :as string]
+            [clojure.tools.logging :as log]
+            [clojure-commons.exception-util :as exception-util]
+            [kameleon.uuids :as uuids]))
+
+(defn- get-paths-exist
+  [user paths]
+  (try+
+   (data-info/get-paths-exist user paths)
+   (catch Object _
+     (log/error (:throwable &throw-context)
+                "job resubmission failed: Could not lookup output folder existence.")
+     (throw+))))
+
+(defn- filter-existing-output-paths
+  [user output-paths]
+  (as-> output-paths p
+    (remove string/blank? p)
+    (get-paths-exist user p)
+    (:paths p)
+    (group-by val p)
+    (get p true)
+    (map first p)))
+
+(defn- find-existing-output-dirs
+  [user jobs]
+  (->> jobs
+       (map :submission)
+       (filter #(not (:create_output_subdir % true)))
```

Comment:
Minor suggestion: you could also use `(remove (:create_output_subdir % true))`

Reply:
Of course! That's cleaner, thanks for the suggestion ✨

Reply:
Oops! I forgot a hash mark in my example. Is should be `(remove #(:create_output_subdir % true))`. Listen to what I mean, not what I type. 😆